contrib/IECoreArnold/src/IECoreArnold/outputDriver/OutputDriver.cpp

```diff
@@ -134,6 +137,12 @@ void driverOpen( AtNode *node, struct AtOutputIterator *iterator, AtBBox2 displa
 			namePrefix = std::string( name ) + ".";
 		}
 
+		const StringData *label = parameters->member< StringData >( "label" );
```

Comment:
I think the problem is that we're mixing labels and layer names here. Dan and I originally agreed via email that we'd pass a layer name, and the layer name for what people think of as "beauty" is "" anyway.